src/libs/fileDownload/index.ios.js

```diff
@@ -0,0 +1,103 @@
+import RNFetchBlob from 'rn-fetch-blob';
+import CameraRoll from '@react-native-community/cameraroll';
+import lodashGet from 'lodash/get';
+import * as FileUtils from './FileUtils';
+import CONST from '../../CONST';
+
+/**
+ * Downloads the file to Documents section in iOS
+ * @param {String} fileUrl
+ * @param {String} fileName
+ * @returns {Promise}
+ */
+function downloadFile(fileUrl, fileName) {
+    const dirs = RNFetchBlob.fs.dirs;
+
+    // The iOS files will download to documents directory
+    const path = dirs.DocumentDir;
+
+    // Fetching the attachment
+    const fetchedAttachment = RNFetchBlob.config({
+        fileCache: true,
+        path: `${path}/${fileName}`,
+        addAndroidDownloads: {
+            useDownloadManager: true,
+            notification: true,
+            path: `${path}/Expensify/${fileName}`,
+        },
+    }).fetch('GET', fileUrl);
+    return fetchedAttachment;
+}
+
+/**
+ * Downloads the image to photo lib in iOS and returns the uri
+ * @param {String} fileUrl
+ * @param {String} fileName
+ * @returns {String}
+ */
+function downloadImage(fileUrl) {
+    return CameraRoll.save(fileUrl);
+}
+
+/**
+ * Downloads the video to photo lib in iOS and returns the uri
+ * @param {String} fileUrl
+ * @param {String} fileName
+ * @returns {String}
+ */
+function downloadVideo(fileUrl, fileName) {
+    return new Promise((resolve) => {
+        let documentPathUri = null;
+        let cameraRollUri = null;
+
+        // Because CameraRoll doesn't allow remote URIs to save the video,
+        // we first download to documents, then copy to photo lib and unlink the original file,
```

Comment:
```suggestion
        // Because CameraRoll doesn't allow direct downloads of video with remote URIs, we first download as documents, then copy to photo lib and unlink the original file.
```